src/test/java/ua/utility/kfsdbupgrade/AppTestBase.java

```diff
@@ -0,0 +1,36 @@
+package ua.utility.kfsdbupgrade;
+
+import java.io.File;
+
+/**
+ * Created by S.G. Skinner on 2017-05-03
+ */
+public abstract class AppTestBase {
+    protected static final String DEFAULT_PROP_FILE =
```

Comment:
Note, for this one, I kept the constant, but it now pulls directly from App.java -- it still needed a base directory, which is what the test constant does.